src/domain/event/occurrences/Occurrences.tsx

```diff
@@ -251,6 +257,27 @@ const Occurrences: React.FC<Props> = ({
               <OccurrenceGroupInfo occurrence={occurrence} />
             </div>
           </div>
+          <div className={styles.infoSection}>
+            <div>
+              <IconGlyphEuro />
+            </div>
+            <div>
+              <div className={styles.infoTitle}>
+                <div data-testid="event-price">
+                  {isFree && t('event:occurrenceList.eventIsFree')}
+                  {!isFree && price}
+                </div>
+              </div>
+              {priceDescription !== '' && (
```

Comment:
would `priceDescription && (...)` be enough?

Reply:
I'm not sure, but I thought the left side needs to evaluated as a boolean or otherwise it is rendered. I've had some issues like that earlier.

Reply:
I think `priceDescription &&` will work here, see: https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator. If priceDescription is string, the should be no problem. Of course little bit safer way is to force left side be boolean with `!!`

`{!!priceDescription && (`